recipes/list_items/contact_info/contact_info_variants.story.vue

```diff
@@ -167,8 +182,28 @@
         </template>
         <template #right>
           <div class="d-d-flex d-ai-center d-m16">
-            <icon-chat class="d-m4" />
-            <icon-menu-horizontal class="d-m4" />
+            <dt-button
+              importance="clear"
+              aria-label="chat"
+            >
+              <template #icon>
+                <dt-icon
+                  name="webchat"
+                  class="d-m4"
+                />
+              </template>
+            </dt-button>
+            <dt-button
+              importance="clear"
+              aria-label="more"
+            >
+              <template #icon>
+                <dt-icon
+                  name="more-horizontal"
+                  class="d-m4"
+                />
+              </template>
+            </dt-button>
```

Comment:
Adding the Tooltip here would be awesome, but I suspect that should be a new issue. Probably requires new prop for the tooltip's contents, and just reusing the `aria-label` wouldn't be enough.

If you agree, log a issue.

Reply:
Yeah, we should create an issue to revisit the styling of the all the examples, not just this one.

Reply:
yep perfect